README.md

```diff
@@ -1,24 +1,57 @@
+This extension can help you write and validate api definition, keys and config files with [VS Code IntelliSense] (https://code.visualstudio.com/docs/editor/intellisense) which validates and autocomplete the required and optional fields and also provides default values and examples to make your work with Tyk faster and easier. This way you can avoid the back&forth copy+amend code snippt from the docs and start using Tyk natively as a language.
 
-## JSON Schemas
+## Installing Tyk Schemas Extension
+Click on the "install" button, like you would with any other extension
+
+## Supported Tyk objects or file formats
 
 For supported file formats and schemas list, check the "Feature Contribution" section in the VS Code extension
 
-All the schemas used by Tyk Schemas extension are taken from [tyk-schemas](https://github.com/tykTechnologies/tyk-schemas) repository. Please refer to it for details on the provided schemas (or to suggest improvements).
+<img width="1244" alt="image" src="https://user-images.githubusercontent.com/3155222/194719459-2b023e04-18ab-4d03-b638-6dea15161d1c.png">
+
+All the schemas used by Tyk Schemas extension are taken from [tyk-schemas](https://github.com/tykTechnologies/tyk-schemas) repository. Please refer to it for details on the provided schemas (or to suggest improvements and report bugs).
+
+The repository that underlies this extention is [vscode-tyk](https://github.com/TykTechnologies/vscode-tyk) - please use it to report bugs or request features such as more file formats. 
+
+
+## Using Tyk Schemas Extension
+
+Once [Tyk Schemas extension](https://marketplace.visualstudio.com/items?itemName=TykTechnologiesLimited.tyk-schemas) is installed, you are ready to start using it. 
+The only requiremnt is to name the file in the following format:
+- `tyk*.conf` - For Tyk gateway config file
+- `apikey*.json`	For tyk gateway keys files/payloads
+- `apidef*.json`, `TykDefinition*.json` - For Tyk API definition files/payloads
+- `oasapidef*.json`, `TykOasApiDef*.json` - For Tyk OAS API definition files/payloads
+
+As you can see below, when you name the file in the right format you'll see a pop up window with suggestion to auto-complete just as it would with any programing language. If you can't see the pop up click `control`+`space`.
+
+<img width="1371" alt="image" src="https://user-images.githubusercontent.com/3155222/194720523-6c6655f7-a12b-4dbe-876b-7d3f8ebd780c.png">
+
 
+### Creating and testing new schemas
 
+In some cases you'd want to add your own schemas or maybe add schemas from our schemas repo that have not been publishe to the VS Code extension just yet.
+In this case you can add the url to the `setting.json` file in VS Code (in MacOS open via "Preferences`), reload the window and start using the schema.
+Example of the code you need to add to the `setting.json` file
 
-## Running Tyk Schemas Extension
+```json
+"json.schemas": [
+        {
+            "fileMatch": [
+                "tyk.*.conf"
```

Comment:
Maybe, we can rename `tyk.*.conf` to something else? because the extension already covers such patterns.
What do you think about renaming `tyk.*.conf` to, let's say, `<your_filename_pattern>`? So that users may understand that `fileMatch` field can be updated based on their preferences?